src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -584,6 +584,7 @@ class RobertaPreTrainedModel(PreTrainedModel):
     config_class = RobertaConfig
     base_model_prefix = "roberta"
     supports_gradient_checkpointing = True
+    _no_split_modules = []
```

Comment:
We don't even need the base block?

Reply:
Yes, for some models (roberta, lilt), passing an empty list was sufficient. I guess the `accelerate` tests are still run since the condition checks only if the list is `None`.
```
if model_class._no_split_modules is None:
    continue
```